styles/widgets/common/buttonGroup.less

```diff
@@ -34,6 +34,9 @@
 
 .dx-buttongroup-item-has-height {
     flex: 1 1 auto;
-    height: 100%;
+
+    :not(.dx-dropdownbutton-toggle) {
```

Comment:
There's a mix of ButtonGroup and DropDownButton classes
If the DropDownButton must somehow change the appearance of the nested ButtonGroup, these styles must be in DropDownButton.*.less